src/NLog.StructuredLogging.Json.Tests/LoggingException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace NLog.StructuredLogging.Json.Tests
+{
+    [Serializable]
```

Comment:
I think I've seen that too, maybe in a previous version of NetStandard?  The lib targets NetStandard 1.3 because [the NLog beta does](https://www.nuget.org/packages/NLog/5.0.0-beta06), but the tests target NetStandard 1.6 because [NUnitLite targets that](https://www.nuget.org/packages/NUnitLite/)